src/Enterspeed.Source.UmbracoCms.Base/NotificationHandlers/EnterspeedContentPublishingNotificationHandler.cs

```diff
@@ -92,30 +95,33 @@ private void ContentServicePublishing(List<IContent> entities)
         /// </summary>
         /// <param name="context"></param>
         /// <param name="content"></param>
-        /// <param name="isPublishConfigured"></param>
         /// <param name="jobs"></param>
-        /// <param name="isPreviewConfigured"></param>
-        private void HandleParentNameChange(UmbracoContextReference context, IContent content, bool isPublishConfigured, ICollection<EnterspeedJob> jobs, bool isPreviewConfigured)
+        private void HandleParentNameChange(UmbracoContextReference context, IContent content, ICollection<EnterspeedJob> jobs)
         {
-            if (context.UmbracoContext.Content != null)
+            if (context.UmbracoContext.Content == null)
             {
-                var currentlyPublishedContent = context.UmbracoContext.Content.GetById(content.Id);
-                var nameHasChanged = !currentlyPublishedContent?.Name?.Equals(content.Name);
```

Comment:
This was line comparing the current culture being saved with the default culture.

Reply:
Good catch :)